test-runner/src/reporters/base.ts

```diff
@@ -98,15 +93,16 @@ export class BaseReporter implements Reporter  {
   epilogue() {
     console.log('');
 
-    console.log(colors.green(`  ${this.passed.length} passed`) + colors.dim(` (${milliseconds(this.duration)})`));
+    console.log(colors.green(`  ${this.asExpected.length} passed`) + colors.dim(` (${milliseconds(this.duration)})`));
 
     if (this.skipped.length)
       console.log(colors.yellow(`  ${this.skipped.length} skipped`));
 
-    if (this.failed.length) {
-      console.log(colors.red(`  ${this.failed.length} failed`));
+    const filteredUnexpected = this.unexpected.filter(t => !t._hasResultWithStatus('timedOut'));
+    if (filteredUnexpected.length) {
+      console.log(colors.red(`  ${filteredUnexpected.length} failed`));
       console.log('');
-      this._printFailures(this.failed);
+      this._printFailures(filteredUnexpected);
```

Comment:
It seems like we print all failures for the test in this line, including timeouts?

Reply:
`const filteredUnexpected = this.unexpected.filter(t => !t._hasResultWithStatus('timedOut'));`